bigtable/admin.go

```diff
@@ -211,18 +211,35 @@ func (ac *AdminClient) Tables(ctx context.Context) ([]string, error) {
 	return names, nil
 }
 
+// DeletionProtection indicates whether the table is protected against data loss
+// i.e. when set to protected, deleting the table, the column families in the table,
+// and the instance containing the table would be prohibited.
+type DeletionProtection int
```

Comment:
A newbie Go question; is the default value of DeletionProtection None? If not, should we make sure it is?

Reply:
I believe yes. The default is the one that has number equals to 0 which is None in our case.

Reply:
+1 to what Kimia said. Default is 0 (None).